xarray/core/dataarray.py

```diff
@@ -3449,18 +3451,13 @@ def quantile(
             Quantile to compute, which must be between 0 and 1 inclusive.
         dim : hashable or sequence of hashable, optional
             Dimension(s) over which to apply quantile.
-        interpolation : {"linear", "lower", "higher", "midpoint", "nearest"}, default: "linear"
+        method : str, default: "linear"
             This optional parameter specifies the interpolation method to
-            use when the desired quantile lies between two data points
-            ``i < j``:
-
-                - linear: ``i + (j - i) * fraction``, where ``fraction`` is
-                  the fractional part of the index surrounded by ``i`` and
-                  ``j``.
-                - lower: ``i``.
-                - higher: ``j``.
-                - nearest: ``i`` or ``j``, whichever is nearest.
-                - midpoint: ``(i + j) / 2``.
+            use when the desired quantile lies between two data points.
+            See numpy.quantile for available methods.
```

Comment:
Is this ok or should we list them explicitly? (The available methods depend on the numpy version which makes this a bit difficult).

Reply:
I would lean toward listing the current available methods but if not, we should make sure `numpy.quantile` get's picked up as a link by intersphinx.